Extension/src/Debugger/attachToProcess.ts

```diff
@@ -108,6 +109,16 @@ export class RemoteAttachPicker {
     private getRemoteProcessCommand(): string {
         let innerQuote: string = `'`;
         let outerQuote: string = `"`;
+        let substitutionBegin: string = `$(`;
+        let substitutionEnd: string = `)`;
+        let escapedQuote: string = `\\\"`;
+
+        let settings: CppSettings = new CppSettings();
```

Comment:
The new setting is "resource" scoped (i.e. per-workspace folder), but no folder uri is passed to CppSettings(). Should the scope of the setting be "window" (i.e. workspace) or is there a way to get the current workspace folder uri?

Reply:
Seems like just making the setting "window" scope would be good enough.

Reply:
This should be per workspace I think

Reply:
window scope is workspace scope (strange naming).